modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_ar.properties

```diff
@@ -15702,6 +15703,7 @@ requested-delivery-date-at-checkout=تاريخ التسليم المطلوب ع
 requested-quantity=الكمية المطلوبة
 requested-report-id=معرِّف التقرير المطلوب
 requested-report-status=حالة التقرير المطلوب
+requesting-results-for-colon=Requesting Results for: (Automatic Copy)
```

Comment:
That's weird. Is `gw buildLang` supposed to add this language that you haven't touched?